### ISSUE TITLE
search is not stored

### ISSUE THREAD

**Issue by Niemand112233** [yuliskov/SmartTubeNext#673]
Hi,

I had to switch from the very good Smarttube to this app, since it does not work anymore. However, STN is sadly not so intuitiv as the old app. 

- the GUI is overloaded
- my search is not stored anymore. I have to type everything everytime again. 

I know you guys are doing a great work, but what can I do to get the old ST experience back?

**Comment by jitunjohn** [comment]
"my search is not stored anymore. I have to type everything everytime again."
I miss the same feature.

**Comment by Niemand112233** [comment]
Yes, It's everytime frustrating using the App because you have to type everything again. And the recommendations are very uninteresting. 

The old App was soooo good. Optical the official YouTube App without adds. This new App is in my opinion just a mess. Sorry to say that, because I can imagine what amount of time you spend on the app. But the old one was, in my opinion, far better.

**Comment by ACR-Jeff** [comment]
This feature is now available in current latest releases.
`Settings > Search > Misc > Disable search history`
If checked then uncheck this option, Search history will then be saved and shown when you enter the search window.
